README.md

```diff
@@ -316,6 +316,44 @@ Note: datasets are first marked for deletion, then are deleted after a certain a
 
 <hr>
 
+## Tags
+Project tags are an easy way to add freeform metadata to a project. Tags help colleagues and consumers organize <br>
+and find projects that interest them in Domino. Tags can be used to describe the subject explored by a project, <br>
+the packages and libraries it uses, or the source of the data within.
+
+### tags_list(*project_id)
+
+returns the list of tags assigned to the specified project
+
+- _project_id:_ id that identifies the specific project to be used
+
+### tag_details(tag_id)
+
+provides detail information about the tag
+
+- _tag_id:_ id that identifies the specific project to be used
+
+### tags_add(tags, *project_id)
+
+Creates a tag, if it does not exist, and add it to the current the project
+
+- _tags (list):_ name that identifies the specific tag to be used
```

Comment:
In addition to the above, I see some examples where the data type is on the right of the `:` in the readme 

 _dataset_ids: (list\[string\])_ 

and other cases where it is not underscored

_runId:_ string

This is fine for now but maybe in the future it would be good to standardize

Reply:
also should it be `names that identify the specific tags to be used` since it is a list

Reply:
moving this section to documents team, and yes.